expression/builtin_math.go

```diff
@@ -852,7 +852,28 @@ type builtinDegreesSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_degrees
 func (b *builtinDegreesSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("degrees")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+
+	sc := b.ctx.GetSessionVars().StmtCtx
+
+	if args[0].IsNull() {
+		d.SetNull()
+		return d, nil
+	}
+
+	tmpIT := sc.IgnoreTruncate
+	sc.IgnoreTruncate = true
+	x, err := args[0].ToFloat64(sc)
+	if err != nil {
+		sc.IgnoreTruncate = tmpIT
```

Comment:
I made a very very wrong choice that I used sqrt built-in function as a reference.